packages/radar/src/hooks.ts

```diff
@@ -83,6 +85,11 @@ export const useRadar = <D extends Record<string, unknown>>({
         color: colorByKey[key],
     }))
 
+    const customLegends = legends.map(legend => ({
```

Comment:
I think this should be memoized, it could lead to unnecessary re-render otherwise.